db/rdb/rdb.go

```diff
@@ -38,11 +39,10 @@ type OvalDB interface {
 	InsertOval(*models.Root, models.FetchMeta, *gorm.DB) error
 }
 
-var ovalMap map[string]OvalDB
+var ovalMap map[string]OvalDB = map[string]OvalDB{}
```

Comment:
`var ovalMap = map[string]OvalDB{}` is better

Reply:
you're right @kotakanbe. Declaration fixed and gofmt passed.